### PR TITLE
Add Chris's Canandaigua file to master

### LINKED ISSUE #7: Create a file for chrisatkins Canandaigua, NY
- [x] Create a branch!!
- [x] Add commits
- [ ] Open Pull Request
- [ ] Collaborate
- [ ] Receive Approval :+1: 
- [ ] Merge :shipit:

### PR DESCRIPTION
**Pull request by chrisatkins**
Closes #7

### CODE REVIEW

File: chrisatkins-canandaigua.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Canandaigua, NY
+#### Places to go
+
+#### Places to Avoid
+
```
Comment (gmasilamani) [inline chrisatkins-canandaigua.md:7]: do you know of places to avoid

File: chrisatkins-canandaigua [hunk 2]
```diff
@@ -0,0 +1,8 @@
+### Canandaigua, NY
+#### Places to go
+
+#### Places to Avoid
+
+#### Places to Eat
+
```
Comment (gmasilamani) [inline chrisatkins-canandaigua]: don't know what to say

**Comment by chrisatkins** [comment]
@khusban So is this code good enough for government work?

**Comment by chrisatkins** [comment]
@thomaslinder do you like pizza

**Pull request merged**